rootfs-builder/rootfs.sh

```diff
@@ -71,6 +72,11 @@ AGENT_INIT          When set to "yes", use ${AGENT_BIN} as init process in place
                     of systemd.
                     Default value: no
 
+AGENT_TRACE         When set to "yes", create a rootfs containing additional
+                    elements to support tracing the agent using https://jaegertracing.io.
+                    Incompatible with AGENT_INIT="yes".
```

Comment:
Why aren't we checking for this in the script?

Reply:
Agree, would be good to add a check to verify that conflicting values have not been set.

Reply:
Good catch - fixed.